azure-pipelines.yml

```diff
@@ -0,0 +1,34 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+pool:
+  vmImage: ubuntu-latest
+
+jobs:
+- job: Test
+  container:
+    image: "linuxbrew/brew"
+  steps:
+  - script: |
+      sudo apt-get update -qq
+      sudo apt-get install -qq automake
+    displayName: Install C++ modules
+  - script: |
+      cd releases/links_v2.0/
+      git submodule init
+      git submodule update
+      ./autogen.sh
+      ./configure
+      make
+    displayName: Compile LINKS
+  - script: |
+      cd releases/links_v2.0/
+      mkdir CI_TEST && cd CI_TEST
+      sudo apt-get install -qq wget
+      wget http://www.bcgsc.ca/downloads/supplementary/LINKS/K12_full2dONT_longread.fa
+      wget http://www.bcgsc.ca/downloads/supplementary/LINKS/ecoliK12_abyss_illumina_contig_baseline.fa
+      echo K12_full2dONT_longread.fa >> K12_F2D.fof
+      ../src/LINKS -f ecoliK12_abyss_illumina_contig_baseline.fa -s K12_F2D.fof -d 4000 -k 15 -t 2 -o 0 -e 0.1 -z 500 -p 0.001 -b CI_TEST -v 1 -x 0
```

Comment:
Not critical for this release, but it would be good to have some code to check that the output of this command is as expected. Also, a test using the makefile.